cvat-ui/src/utils/conflict-detector.ts

```diff
@@ -161,3 +161,36 @@ export function unsetExistingShortcuts(
     }
     registerComponentShortcuts(updatedShortcuts);
 }
+
+function removeConflictingSequences(
+    sequences: string[],
+    conflictingSequences: string[],
+): string[] {
+    const isConflict = (sequence: string) => (conflictingSequence: string) => conflict(sequence, conflictingSequence);
+    const nonConflictingSequence = (sequence: string): boolean => (
+        !conflictingSequences.some(isConflict(sequence))
+    );
+
+    return sequences.filter(nonConflictingSequence);
+}
+
+export function resolveConflicts(
+    updateKeyMap: KeyMap,
+    shortcutsKeyMap: KeyMap,
+): KeyMap {
+    const resultMap = cloneDeep(updateKeyMap);
+
+    Object.entries(resultMap).forEach(([key, currValue]) => {
+        const conflicts = conflictDetector({ [key]: currValue }, shortcutsKeyMap);
+        if (!conflicts) return;
+
+        Object.keys(conflicts).forEach((conflictingKey) => {
+            resultMap[conflictingKey].sequences = removeConflictingSequences(
+                resultMap[conflictingKey].sequences,
+                conflicts[conflictingKey].sequences,
+            );
+        });
+    });
```

Comment:
Let's consider:

- wrongShortcut to be a shortcut which has conflicts with any other shortcuts
- conflictingShortcut to be a shortcut which conflicts with wrong shortcut

E.g. I set `TOGGLE_ANNOTATION_PAGE` to `f1` in local storage
And we have `SWITCH_SHORTCUTS` also having `f1` sequence

`TOGGLE_ANNOTATION_PAGE` in this scheme is `wrongShortcut`
`SWITCH_SHORTCUTS` is conflicting shortcut

The problem I see: we are filtering sequences of the conflictingShortcut (SWITCH_SHORTCUTS), however it would be more correct to filter sequences of wrongShortcut (TOGGLE_ANNOTATION_PAGE)